modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/actionItems/formatActionURL.ts

```diff
@@ -37,12 +39,45 @@ const formatActionURL = function (url: string | undefined, item: any): string {
 		)
 	);
 
-	return replacedURL.replace(new RegExp('(%7B.*?%7D)', 'mg'), (matched) =>
+	replacedURL.replace(new RegExp('(%7B.*?%7D)', 'mg'), (matched) =>
 		getValueFromItem(
 			item,
 			matched.substring(3, matched.length - 3).split('.')
 		)
 	);
+
+	if (replacedURL.includes('?')) {
+		const redirectionURL = window.location.href;
+		const backURL = '_backURL';
+		const redirect = '_redirect';
+		const backURLRegexp = new RegExp(backURL);
+		const redirectRegexp = new RegExp(redirect);
+
+		const searchParams = new URLSearchParams(
```

Comment:
Out of curiosity https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams